providers/tentacle.rb

```diff
@@ -63,8 +63,8 @@
   config_path = new_resource.config_path
   app_path = new_resource.app_path
   trusted_cert = new_resource.trusted_cert
-  port = new_resource.port
   polling = fancy_bool(new_resource.polling)
```

Comment:
I would only use fancy_bool in powershell scripts.  you can use that function with the `#{fancy_bool(polling)}` I am pretty sure